lib/Log/Dispatch/Gelf.pm

```diff
@@ -36,6 +37,28 @@ sub _init {
             additional_fields => { type => HASHREF, optional => 1 },
             host              => { type => SCALAR,  optional => 1 },
             compress          => { type => BOOLEAN, optional => 1 },
+            chunked           => {
+                type => SCALAR,
+                optional => 1,
+                callbacks => {
+                    check_valid_size => sub {
+                        my ($chunked) = @_;
+
+                        die 'chunked must be "wan", "lan", or a positive integer'
+                            unless $chunked =~ /^(wan|lan|\d+)$/i;
+
+                        if ( lc($1) eq 'wan' ) {
+                            $self->{chunked} = 1420;
+                        }
+                        elsif ( lc($1) eq 'lan' ) {
+                            $self->{chunked} = 8154;
```

Comment:
Why 8154B? They say 8192B in the documentation (http://docs.graylog.org/en/latest/pages/gelf.html#chunking) - headers 12B = 8180B. What are the remaining 26B for? Could you please comment on this in the code or introduce some self-explanatory constants?